solr/contrib/ltr/README.md

```diff
@@ -285,17 +284,17 @@ libSVM model format to the format specified above.
     }
 }
 ```
-This is an example of a toy LambdaMART. Type specifies the class to be using to
-interpret the model (LambdaMARTModel in the case of LambdaMART). Name is the
+This is an example of a toy Multiple Additive Trees. Type specifies the class to be using to
```

Comment:
We should replace 'type' with class in the readme.  There might be other locations we missed.